docker/drama-free-django/_test.sh

```diff
@@ -58,7 +60,8 @@ grep -Eho \
     | xargs touch
 
 # Now that we're sure that we have webfont files, we can test collectstatic.
-./venv/bin/django-admin collectstatic
+#./venv/bin/django-admin collectstatic
```

Comment:
Probably don't need this commented out code here.

Reply:
fixed!